pipelines/shopgate.user.addFavorites.v1.json

```diff
@@ -0,0 +1,85 @@
+{
+  "version": "1",
+  "pipeline": {
+    "id": "shopgate.user.addFavorites.v1",
+    "public": true,
+    "input": [
+      {"key": "sgxsMeta", "id": "16"},
+      {"key": "productId", "id": "1"}
+    ],
+    "output": [],
+    "steps": [
+      {
+        "type": "extension",
+        "id": "@shopgate/favorites",
+        "path": "@shopgate/favorites/convertToArray.js",
+        "input": [
+          {"key": "productId", "id": "1"}
+        ],
+        "output": [
+          {"key": "productIds", "id": "1"}
```

Comment:
Key is not that important but why is it singular in input but plural for the output?!